check_metapackage.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Author: Yuki Furuta <furushchev@jsk.imi.i.u-tokyo.ac.jp>
```

Comment:
This file should contain a license header.  Since the author is not @130s, we must make sure we use the whatever license file is with the original file.  I realize this is a trivial source file, but at the very least we should give credit where credit is due.